pkg/controllers/apiextensions/setup.go

```diff
@@ -30,27 +30,26 @@ import (
 const (
 	maxConcurrency = 5
 
-	infraPubCRDName    = "infrastructurepublications.apiextensions.crossplane.io"
-	infraDefCRDName    = "infrastructuredefinitions.apiextensions.crossplane.io"
+	xrdCRDName         = "compositeresourcedefinition.apiextensions.crossplane.io"
```

Comment:
```suggestion
	xrdCRDName         = "compositeresourcedefinitions.apiextensions.crossplane.io"
```

I think this would be plural, right?